### ISSUE TITLE
[AUTOMATIC] Update Polkadot dependencies from 1.1.0 to 1.2.0

### ISSUE THREAD

**Issue by kiltbot** [KILTprotocol/kilt-node#571]
### Track updates
```
repository: paritytech/polkadot-sdk
commits   : 131
command   : subalfred track-updates paritytech/polkadot-sdk --from polkadot-v1.1.0 --to polkadot-v1.2.0
```
> https://github.com/paritytech/polkadot-sdk/compare/polkadot-v1.1.0...polkadot-v1.2.0
### All
- [Fix PRdoc check](https://github.com/paritytech/polkadot-sdk/pull/1419) - `R0-silent`
- [Remove deprecated `pallet_balances`'s `set_balance_deprecated` and `transfer` dispatchables](https://github.com/paritytech/polkadot-sdk/pull/1226) - `T2-pallets`, `T10-polkadot`
- [pallet asset-conversion additional quote tests](https://github.com/paritytech/polkadot-sdk/pull/1371) - `T12-tests`
- [GHW for building and publishing docker images](https://github.com/paritytech/polkadot-sdk/pull/1391) - `R0-silent`
- [Fix nothing scheduled on session boundary](https://github.com/paritytech/polkadot-sdk/pull/1403) - `T10-polkadot`
- [Prevent a fail prdoc check to block](https://github.com/paritytech/polkadot-sdk/pull/1433) - `R0-silent`, `T13-documentation`
- [zombienet: use another collator image for the slashing test](https://github.com/paritytech/polkadot-sdk/pull/1386) - `T12-tests`
- [Adds base benchmark for do_tick in broker pallet](https://github.com/paritytech/polkadot-sdk/pull/1235) - `I2-bug`, `T2-pallets`, `T14-benchmarks`
- [Forgotten `polkadot-core-primitives/std`](https://github.com/paritytech/polkadot-sdk/pull/1440) - `T10-polkadot`
- [remove unused `keystore_uri`](https://github.com/paritytech/polkadot-sdk/pull/1421) - `T0-node`
- [[ci] Enable flacky collector for tests](https://github.com/paritytech/polkadot-sdk/pull/1439)
- [polkadot: pin one block per session](https://github.com/paritytech/polkadot-sdk/pull/1220) - `R0-silent`, `T0-node`, `T8-parachains_engineering`
- [[ci] Return publish-rustdoc](https://github.com/paritytech/polkadot-sdk/pull/1402) - `R0-silent`
- [[ci] change image for prdoc gha](https://github.com/paritytech/polkadot-sdk/pull/1452) - `R0-silent`
- [Update Chrono to 0.4.30](https://github.com/paritytech/polkadot-sdk/pull/1451) - `R0-silent`
- [Bump actions/cache from 3.3.1 to 3.3.2](https://github.com/paritytech/polkadot-sdk/pull/1456) - `A1-insubstantial`, `R0-silent`
- [Bump hex-literal from 0.3.4 to 0.4.1](https://github.com/paritytech/polkadot-sdk/pull/1438) - `A1-insubstantial`, `R0-silent`
- [Update ed25519-dalek to v2](https://github.com/paritytech/polkadot-sdk/pull/1446) - `R0-silent`, `T12-tests`
- [Switch prdoc to the latest image](https://github.com/paritytech/polkadot-sdk/pull/1460) - `R0-silent`, `T13-documentation`
- [RuntimeCode: doc fixed](https://github.com/paritytech/polkadot-sdk/pull/1461) - `T13-documentation`
- [FRAME: Consideration (abstraction over storage deposits)](https://github.com/paritytech/polkadot-sdk/pull/1361) - `T1-FRAME`
- [lookahead collator: read allowed ancestry len from relay client](https://github.com/paritytech/polkadot-sdk/pull/1180)
- [Use sp_std::vec::Vec in sp-crypto-ec-utils](https://github.com/paritytech/polkadot-sdk/pull/1471) - `T0-node`
- [Handle `CollationSeconded` Logs Conditionally](https://github.com/paritytech/polkadot-sdk/pull/1475) - `T9-parachains_protocol`
- [Introduce `default-members` to the workspace root](https://github.com/paritytech/polkadot-sdk/pull/1420) - `T0-node`
- [Remove `SafeCallFilter` from Relay Runtimes](https://github.com/paritytech/polkadot-sdk/pull/1303) - `T10-polkadot`
- [Update bridges subtree](https://github.com/paritytech/polkadot-sdk/pull/1392) - `A3-backport`, `T17-bridges`
- [Remove redundant book keeping in dispute coordinator.](https://github.com/paritytech/polkadot-sdk/pull/1432) - `T10-polkadot`
- [Merge `CODEOWNERS` Part 1](https://github.com/paritytech/polkadot-sdk/pull/1472) - `R0-silent`
- [Set bridge crates to no-publish](https://github.com/paritytech/polkadot-sdk/pull/1488) - `A1-insubstantial`
- [Bump the known_good_semver group with 2 updates](https://github.com/paritytech/polkadot-sdk/pull/1485) - `A1-insubstantial`, `R0-silent`
- [[ci] Move additional tests back to test-linux-stable](https://github.com/paritytech/polkadot-sdk/pull/1466) - `R0-silent`
- [[improve docs]: Sudo Pallet](https://github.com/paritytech/polkadot-sdk/pull/1209) - `A1-insubstantial`, `R0-silent`, `T1-FRAME`, `T13-documentation`
- [Delete staking miner](https://github.com/paritytech/polkadot-sdk/pull/1480) - `T10-polkadot`
- [Retire puppet workers](https://github.com/paritytech/polkadot-sdk/pull/1449) - `T12-tests`
- [Allow to broadcast network messages in parallel](https://github.com/paritytech/polkadot-sdk/pull/1409) - `A0-needs_burnin`, `T0-node`, `A3-backport`, `I4-refactor`, `T10-polkadot`
- [contracts: Run start function](https://github.com/paritytech/polkadot-sdk/pull/1367) - `T2-pallets`
- [fix simple analytics injection script](https://github.com/paritytech/polkadot-sdk/pull/1501) - `A1-insubstantial`, `R0-silent`
- [Additional logging for `dispute-coordinator`](https://github.com/paritytech/polkadot-sdk/pull/1494) - `T8-parachains_engineering`
- [Bump cfg-expr from 0.15.4 to 0.15.5](https://github.com/paritytech/polkadot-sdk/pull/1502) - `A1-insubstantial`, `R0-silent`
- [Fix mistakes in Cargo.toml](https://github.com/paritytech/polkadot-sdk/pull/1504) - `A1-insubstantial`, `R0-silent`
- [Report `tracing_unbounded` channel size to prometheus](https://github.com/paritytech/polkadot-sdk/pull/1489) - `T0-node`
- [[CI] Disable runtime logging for benchmarks](https://github.com/paritytech/polkadot-sdk/pull/1463) - `T0-node`, `T4-runtime_API`
- [Make the node version independent of the crate version](https://github.com/paritytech/polkadot-sdk/pull/1495) - `T0-node`
- [add tests for `find_potential_parents`](https://github.com/paritytech/polkadot-sdk/pull/1338) - `T11-cumulus`, `T12-tests`
- [lookahead collator: only build on top of scheduled relay parents](https://github.com/paritytech/polkadot-sdk/pull/1429) - `T11-cumulus`
- [Allow justifications on non-finalized blocks](https://github.com/paritytech/polkadot-sdk/pull/1211) - `T0-node`
- [Oleg/ci cd/add new tag](https://github.com/paritytech/polkadot-sdk/pull/1427) - `R0-silent`
- [pallet-beefy-mmr: better logging on BEEFY key to ETH address conversion](https://github.com/paritytech/polkadot-sdk/pull/1520) - `T2-pallets`
- [Stabilize `VersionedMigration`](https://github.com/paritytech/polkadot-sdk/pull/1503) - `T1-FRAME`
- [Bump the known_good_semver group with 1 update](https://github.com/paritytech/polkadot-sdk/pull/1530) - `A1-insubstantial`, `R0-silent`
- [fix(ci): Add logic to check and calculate secundary image to use](https://github.com/paritytech/polkadot-sdk/pull/1529) - `A1-insubstantial`, `R0-silent`, `T12-tests`
- [Fix PRCR's Zombienet `exclude:` behavior](https://github.com/paritytech/polkadot-sdk/pull/1535) - `R0-silent`
- [Inclusions struct unit tests](https://github.com/paritytech/polkadot-sdk/pull/1518) - `T8-parachains_engineering`, `T12-tests`
- [Reintroduce and fix Docker image build for production](https://github.com/paritytech/polkadot-sdk/pull/1536) - `R0-silent`, `T0-node`
- [Update trie-db version to 0.28.0](https://github.com/paritytech/polkadot-sdk/pull/1522) - `A1-insubstantial`, `R0-silent`
- [Bandersnatch tweaks after backend update](https://github.com/paritytech/polkadot-sdk/pull/1482) - `T0-node`
- [wasm-builder: Disable building when running on docs.rs](https://github.com/paritytech/polkadot-sdk/pull/1540) - `T0-node`
- [Add tolerance to nom pool pending rewards try-state](https://github.com/paritytech/polkadot-sdk/pull/1236) - `T1-FRAME`
- [frame/beefy: add privileged call to reset BEEFY consensus](https://github.com/paritytech/polkadot-sdk/pull/1534) - `T2-pallets`
- [[xcm_builder]: Do not consume `msg` on `NotApplicable` for remote exporters](https://github.com/paritytech/polkadot-sdk/pull/1519) - `T6-XCM`
- [Enable `runtime-benchmarks` feature for crates](https://github.com/paritytech/polkadot-sdk/pull/1544) - `T11-cumulus`
- [Inclusion pruning tweaks](https://github.com/paritytech/polkadot-sdk/pull/1550) - `T8-parachains_engineering`, `T12-tests`
- [Fix code duplication in tests](https://github.com/paritytech/polkadot-sdk/pull/1575) - `T8-parachains_engineering`
- [upgrade to docify v0.2.3](https://github.com/paritytech/polkadot-sdk/pull/1555) - `A1-insubstantial`, `R0-silent`, `T1-FRAME`
- [Modular block request handler](https://github.com/paritytech/polkadot-sdk/pull/1524) - `T0-node`
- [Added `xcmp_queue_send_xcm_works` test](https://github.com/paritytech/polkadot-sdk/pull/1422) - `T2-pallets`, `T11-cumulus`
- [Bump the known_good_semver group with 2 updates](https://github.com/paritytech/polkadot-sdk/pull/1553) - `A1-insubstantial`, `R0-silent`
- [Enable publishing of crates that are deps of other published crates](https://github.com/paritytech/polkadot-sdk/pull/1573) - `A1-insubstantial`, `R0-silent`
- [asset-rate pallet: box asset kind parameter](https://github.com/paritytech/polkadot-sdk/pull/1545) - `T1-FRAME`, `T2-pallets`, `T4-runtime_API`
- [rpc-v2: Change method name for provided events](https://github.com/paritytech/polkadot-sdk/pull/1593) - `A0-needs_burnin`, `R0-silent`, `D0-easy`
- [Include `bitvec` in `std` for Broker Pallet](https://github.com/paritytech/polkadot-sdk/pull/1558) - `A1-insubstantial`, `T1-FRAME`, `T2-pallets`
- [FRAME: Revamp Preimage pallet to use Consideration](https://github.com/paritytech/polkadot-sdk/pull/1363) - `T2-pallets`
- [Babe epoch newtype](https://github.com/paritytech/polkadot-sdk/pull/1596) - `T0-node`
- [fix: export-genesis-state command](https://github.com/paritytech/polkadot-sdk/pull/1521) - `T0-node`
- [[Backport] Version bumps from release 1.1.0](https://github.com/paritytech/polkadot-sdk/pull/1580) - `R0-silent`, `A3-backport`
- [Executor: Remove `LegacyInstanceReuse` strategy](https://github.com/paritytech/polkadot-sdk/pull/1486) - `T0-node`
- [xcm-builder: PayOverXcm supports fallible convertors for asset kind and beneficiary conversion ](https://github.com/paritytech/polkadot-sdk/pull/1572) - `T6-XCM`, `T4-runtime_API`
- [[improve docs] Example pallet crate and Basic Example pallet](https://github.com/paritytech/polkadot-sdk/pull/1546) - `T13-documentation`
- [[improve docs]: Timestamp pallet](https://github.com/paritytech/polkadot-sdk/pull/1435) - `T13-documentation`
- ["Common good" vs "System" parachain clean up](https://github.com/paritytech/polkadot-sdk/pull/1406) - `T10-polkadot`, `T11-cumulus`
- [Implements a variable deposit base calculation for EPM signed submissions](https://github.com/paritytech/polkadot-sdk/pull/1547) - `T1-FRAME`, `T10-polkadot`
- [Bump docker/setup-buildx-action from 2.1.0 to 3.0.0](https://github.com/paritytech/polkadot-sdk/pull/1551) - `A1-insubstantial`, `R0-silent`
- [Replace secrets with the new ones](https://github.com/paritytech/polkadot-sdk/pull/1564) - `R0-silent`
- [Broker pallet: `RegionDropped` event fix & additional tests](https://github.com/paritytech/polkadot-sdk/pull/1609) - `R0-silent`
- [Bump docker/login-action from 2 to 3](https://github.com/paritytech/polkadot-sdk/pull/1531) - `A1-insubstantial`, `R0-silent`
- [Bump docker/build-push-action from 4 to 5](https://github.com/paritytech/polkadot-sdk/pull/1552) - `A1-insubstantial`, `R0-silent`
- [ chainHead: Add support for storage closest merkle descendant #14818](https://github.com/paritytech/polkadot-sdk/pull/1153) - `A0-needs_burnin`, `R0-silent`
- [Bump the known_good_semver group with 1 update](https://github.com/paritytech/polkadot-sdk/pull/1606) - `A1-insubstantial`, `R0-silent`
- [Move ISSUE_TEMPLATE](https://github.com/paritytech/polkadot-sdk/pull/1567) - `R0-silent`
- [[ci] Publish implementers guide](https://github.com/paritytech/polkadot-sdk/pull/1615) - `R0-silent`
- [Replace free for all collation in `cumulus` runtimes](https://github.com/paritytech/polkadot-sdk/pull/1251) - `A0-needs_burnin`, `I9-optimisation`, `T9-parachains_protocol`
- [Staking: Add `dest` to `Rewarded` to aid in reward calculations](https://github.com/paritytech/polkadot-sdk/pull/1602) - `I5-enhancement`, `T2-pallets`
- [Revert #1409 partially](https://github.com/paritytech/polkadot-sdk/pull/1603) - `A0-needs_burnin`, `T0-node`, `I2-bug`, `D0-easy`
- [Remove Polkadot & Kusama native runtime](https://github.com/paritytech/polkadot-sdk/pull/1304) - `T10-polkadot`
- [archive: Fetch body, genesisHash and header](https://github.com/paritytech/polkadot-sdk/pull/1560) - `A0-needs_burnin`, `I5-enhancement`, `T3-RPC_API`, `D0-easy`
- [Add Method to Establish HRMP Channels Among System Parachains](https://github.com/paritytech/polkadot-sdk/pull/1473) - `T10-polkadot`
- [allow governance body on parachain to have sovereign account](https://github.com/paritytech/polkadot-sdk/pull/1291) - `T10-polkadot`
- [Disable Calls to Identity Pallet](https://github.com/paritytech/polkadot-sdk/pull/1476) - `T10-polkadot`
- [Refactor availability-recovery strategies](https://github.com/paritytech/polkadot-sdk/pull/1457) - `A0-needs_burnin`, `T0-node`, `I4-refactor`
- [XCM: Deprecate old functions](https://github.com/paritytech/polkadot-sdk/pull/1645) - `T6-XCM`
- [Bump the known_good_semver group with 2 updates](https://github.com/paritytech/polkadot-sdk/pull/1620) - `A1-insubstantial`, `R0-silent`
- [Preimage: Check that at least one is upgraded](https://github.com/paritytech/polkadot-sdk/pull/1648) - `T2-pallets`
- [Fix minor typos and file name in contributing docs](https://github.com/paritytech/polkadot-sdk/pull/1651) - `A1-insubstantial`, `T13-documentation`
- [upgrade to docify v0.2.4](https://github.com/paritytech/polkadot-sdk/pull/1653) - `R0-silent`, `T1-FRAME`
- [The Ambassador Program](https://github.com/paritytech/polkadot-sdk/pull/1308) - `T16-system_parachains`
- [Add graceful shutdown to prometheus server](https://github.com/paritytech/polkadot-sdk/pull/1637) - `R0-silent`
- [bump zombienet version](https://github.com/paritytech/polkadot-sdk/pull/1655) - `A1-insubstantial`, `R0-silent`, `T12-tests`, `D0-easy`
- [[Trivial] Use 'into_inner' for WeakBoundedVec](https://github.com/paritytech/polkadot-sdk/pull/1664) - `T0-node`
- [Bump quinn-proto from 0.9.4 to 0.9.5](https://github.com/paritytech/polkadot-sdk/pull/1670) - `A1-insubstantial`, `R0-silent`
- [Use correct target dir in polkadot readme](https://github.com/paritytech/polkadot-sdk/pull/1643) - `R0-silent`, `T10-polkadot`
- [update `genesis_config` related docs and tests and error messages](https://github.com/paritytech/polkadot-sdk/pull/1642) - `T13-documentation`
- [Update HRMP pallet benchmarking to use benchmarks v2](https://github.com/paritytech/polkadot-sdk/pull/1676) - `T2-pallets`, `T14-benchmarks`, `T16-system_parachains`
- [pallet epm: add `TrimmingStatus` to the mined solution](https://github.com/paritytech/polkadot-sdk/pull/1659) - `T2-pallets`
- [update contributing guide and ui-tests scripts](https://github.com/paritytech/polkadot-sdk/pull/1668) - `R0-silent`, `T13-documentation`
- [docs / Update PR template to reflect monorepo](https://github.com/paritytech/polkadot-sdk/pull/1674) - `T13-documentation`
- [Bump aes-gcm from 0.10.2 to 0.10.3](https://github.com/paritytech/polkadot-sdk/pull/1681) - `A1-insubstantial`, `R0-silent`
- [Uncoupling pallet-xcm from frame-system's RuntimeCall](https://github.com/paritytech/polkadot-sdk/pull/1684)
- [tweak pallet macro (genesis_config etc) to cater for RA users as well.](https://github.com/paritytech/polkadot-sdk/pull/1689) - `R0-silent`
- [Fix documentation about justification and `finalized == true` requirement](https://github.com/paritytech/polkadot-sdk/pull/1607) - `R0-silent`
- [contracts: Fix incorrect storage alias in mirgration](https://github.com/paritytech/polkadot-sdk/pull/1687) - `R0-silent`, `I2-bug`, `T2-pallets`, `T7-smart_contracts`, `D0-easy`
- [Bump actions/checkout from 4.0.0 to 4.1.0](https://github.com/paritytech/polkadot-sdk/pull/1688) - `A1-insubstantial`, `R0-silent`
- [Make downloads in parallel and give more time to complete](https://github.com/paritytech/polkadot-sdk/pull/1699) - `A1-insubstantial`, `R0-silent`, `T12-tests`
- [Implement more useful traits in `Slot` type](https://github.com/paritytech/polkadot-sdk/pull/1595) - `R0-silent`
- [chainHead/storage: Fix storage iteration using the query key](https://github.com/paritytech/polkadot-sdk/pull/1665) - `A1-insubstantial`, `I2-bug`, `D1-medium`
- [Allow debug_assertions in short-benchmarks CI job](https://github.com/paritytech/polkadot-sdk/pull/1711) - `T12-tests`, `T14-benchmarks`
- [Bump directories from 4.0.1 to 5.0.1](https://github.com/paritytech/polkadot-sdk/pull/1656) - `A1-insubstantial`, `R0-silent`
- [`BlockId` removal: `tx-pool` refactor](https://github.com/paritytech/polkadot-sdk/pull/1678) - `T0-node`
- [genesis-builder: implemented for all runtimes](https://github.com/paritytech/polkadot-sdk/pull/1492) - `T4-runtime_API`
- [Migrate polkadot-primitives to v6](https://github.com/paritytech/polkadot-sdk/pull/1543) - `T4-runtime_API`, `T10-polkadot`
- [Associated type Hasher for `QueryPreimage`, `StorePreimage` and `Bounded`](https://github.com/paritytech/polkadot-sdk/pull/1720) - `T1-FRAME`
### Watched labels

**Comment by ntn-x2** [comment]
Duplicate of https://github.com/KILTprotocol/ticket/issues/3434.
